apps/complaints/translations/src/en/buttons.json

```diff
@@ -0,0 +1,3 @@
+{
+  "continue": "Save and continue"
```

Comment:
You are correct. Just checked with UX should be 'Continue'.
Prototype was slightly wrong.